scripts/README.md

```diff
@@ -0,0 +1,217 @@
+# Firebase Apple Scripts
+
+This directory provides a set of scripts for development, test, and continuous
+integration of the Firebase Apple SDKs.
+
+## [check.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/check.sh)
+
+Used by the
+[check CI workflow](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/check.yml)
+to run several static analysis checks. It calls the following scripts:
+
+### [style.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/style.sh)
+
+Runs clang-format and swiftformat across the repo.
+
+### [check_whitespace.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/check_whitespace.sh)
+
+Verify there are files with trailing whitespace.
```

Comment:
nit: This should say `no files with trailing whitespace`.

Reply:
Fixed